src/indicators/FoodIndicator.js

```diff
@@ -22,13 +22,15 @@ export const FoodIndicator = ({
   setFoodPercent,
   stateIndicator,
 }) => {
+  const intervalTime = 500;
+  const condition_for_clearUseInterval =
```

Comment:
mixing camelCase with snake_case is not a good naming practice

Reply:
+